boards/arm/disco_l475_iot1/disco_l475_iot1_defconfig

```diff
@@ -5,6 +5,7 @@ CONFIG_BOARD_DISCO_L475_IOT1=y
 CONFIG_SOC_SERIES_STM32L4X=y
 CONFIG_SOC_STM32L475XX=y
 CONFIG_CORTEX_M_SYSTICK=y
+CONFIG_CLOCK_STM32_SYSCLK_SRC_PLL=y
```

Comment:
Move this one down to Clock section.

Reply:
Also we'll need to remove :
```
CONFIG_CLOCK_STM32_LSE=y
CONFIG_CLOCK_STM32_MSI_PLL_MODE=y
```
That are no more required since MSI is not used.

Reply:
Updated, does this looks better to you ?